full-service/src/db/txo.rs

```diff
@@ -3021,6 +3026,7 @@ mod tests {
             None,
             0,
             Mob::MINIMUM_FEE,
+            None,
```

Comment:
These tests do not cover the `only_from_subaddress_b58` so they all set the new field to None to test functionality from listing/spending from the main account (and sourcing from any txo no matter which subaddress it came through)